swagger.yaml

```
@@ -4407,6 +4562,58 @@ components:
           type: string
           example: "1a3be38bcbb7911969283716ad7aa550250226b76a61fc51cc9a9a35d9276d81"
           description: Epoch number only used once
+        # cost_models:
+        # https://github.com/input-output-hk/cardano-db-sync/pull/758
```

```suggestion
        cost_models:
          nullable: true
          type: object
```

This is not fixed in the latest release, but maybe we could return `null` in the meantime, just to introduce the field. It's `nullable` anyway.

---

On a second thought, let's wait since there's a pending underlying schema change.